examples/webcam/webcam.py

```diff
@@ -9,11 +9,34 @@
 from aiohttp import web
 
 from aiortc import RTCPeerConnection, RTCSessionDescription
-from aiortc.contrib.media import MediaPlayer
+from aiortc.contrib.media import MediaBroadcaster, MediaPlayer
 
 ROOT = os.path.dirname(__file__)
 
 
+broadcaster = None
+webcam = None
+
+
+def create_local_tracks(play_from):
+    global broadcaster, webcam
+
+    if play_from:
+        player = MediaPlayer(play_from)
+        return player.audio, player.video
```

Comment:
This will create a mediaplayer for each individul (starting the media source from the beginning), however I could image you want to "sync" the received media for all receiving clients

Reply:
This is on purpose: when playing from a file, we want each subscriber to start at the beginning of the file => using a MediaBroadcaster does not make sense. The only type of source for which it makes sense is for "live" sources which you can only acquire once:

- a webcam or other capture device
- a track received from a remote party

Reply:
Adding it as a arg option to create a "semi-live" media file source?.